ddl/ddl.go

```diff
@@ -134,7 +134,7 @@ type DDL interface {
 	// GetLease returns current schema lease time.
 	GetLease() time.Duration
 	// Stats returns the DDL statistics.
-	Stats() (map[string]interface{}, error)
+	Stats(vars *variable.SessionVars) (map[string]interface{}, error)
```

Comment:
`SHOW STATUS` may contain session specific status variables (like Ssl_cipher in this PR). Previously we only support global status variables. I don't think this solution perfect as well. Do you have some better ideas?